monkey/tests/unit_tests/common/agent_events/test_file_encryption_event.py

```diff
@@ -3,6 +3,7 @@
 from uuid import UUID
 
 import pytest
+from pydantic.errors import IntegerError
```

Comment:
The fact that we're using pydantic shouldn't be leaked anywhere outside of the models. We should catch this and raise a custom exception.